Automatic Validation ended with:
> 2024-04-11T16:31:37.6984313Z ##[error]Manifest Validation Failed
 2024-04-11T16:31:42.7479250Z ##[section]Finishing: Validate Manifest

(Automated response - build 876.)

--- a/manifests/o/OpenRCT2/OpenRCT2/0.4.10/OpenRCT2.OpenRCT2.installer.yaml
+++ b/manifests/o/OpenRCT2/OpenRCT2/0.4.10/OpenRCT2.OpenRCT2.installer.yaml
@@ -11,7 +11,7 @@ InstallerSuccessCodes:
 UpgradeBehavior: install
 ProductCode: OpenRCT2
 AppsAndFeaturesEntries:
-- DisplayVersion: 0.4.9
+- DisplayVersion: 0.4.10
   ProductCode: OpenRCT2
 Installers:
 - Architecture: x86

